### PR TITLE
Fix issue #53 - add slash at the end of path prefix

### PR DESCRIPTION
**Pull request by alexfoias**
To test use the procedure explained [here](https://github.com/neuropoly/sct_testing_management#3-run-tests-from-your-machine).

For ssh tunnelling use local generic account on joplin.

### CODE REVIEW

**Comment by alexfoias** [comment]
@jcohenadad Can I merge this pull request ?

**Comment by jcohenadad** [comment]
@alexfoias i've added a comment (unaddressed)

File: sct_annotation/annotations/models.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class FileNameMixin(models.Model):
         'File path',
         max_length=512,
         unique=True,
-        help_text=f'path prefix: {settings.SCT_DATASET_ROOT}'
+        help_text=f'path prefix: {settings.SCT_DATASET_ROOT}/'
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply sct_annotation/annotations/models.py]: hum... i think `SCT_DATASET_ROOT` refers to the physical path of sct_testing mounted on tristano (i.e. where the database accesses the data) so if this is the correct path, then i would leave it like that.

[the remark this answers is not in the document]
Comment (alexfoias) [reply sct_annotation/annotations/models.py]: SCT_DATASET_ROOT is just an alias for the mountpoint on tristano. My suggestion is just to change the help text which is displayed on the form.

[the remark this answers is not in the document]
Comment (jcohenadad) [reply sct_annotation/annotations/models.py]: yes, that's what i meant (mountpoint on tristano, which is where the DB accesses the data). I don't understand why you want to change this path?

[the remark this answers is not in the document]
Comment (alexfoias) [reply sct_annotation/annotations/models.py]: I was think that it with help user to retrace the data under duke, but we can keep it to the current pattern.

[the remark this answers is not in the document]
Comment (jcohenadad) [reply sct_annotation/annotations/models.py]: depending on where the user is mounting `sct_testing`, the path could change. So in other words, `/duke/sct_testing/large` does not work for everyone, and therefore it should not be used.

[the remark this answers is not in the document]
Comment (alexfoias) [reply sct_annotation/annotations/models.py]: I totally agree your point. I think we can proceed with the current version.

**Pull request merged**